src/tests/functional/plugin/shared/src/behavior/compiled_model/properties.cpp

```diff
@@ -255,6 +257,17 @@ TEST_P(OVClassCompiledModelGetPropertyTest, GetMetricNoThrow_OPTIMAL_NUMBER_OF_I
     ASSERT_EXEC_METRIC_SUPPORTED(ov::optimal_number_of_infer_requests);
 }
 
+TEST_P(OVClassCompiledModelGetPropertyTest, CanCompileModelWithEmptyProperties) {
+    ov::Core core = createCoreWithTemplate();
+
+    OV_ASSERT_NO_THROW(core.compile_model(simpleNetwork, target_device, ov::AnyMap{}));
+}
+
+TEST_P(OVClassCompiledModelGetPropertyTest, CompileModelWithBigDeviceIDThrows) {
```

Comment:
we have already this test on str 120 : (OVCompiledModelIncorrectDevice, CanNotCompileModelWithIncorrectDeviceID)